docs/conf.py

```diff
@@ -162,16 +162,47 @@ def get_version(*file_paths):
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 
-html_theme = 'edx_theme'
+html_theme = 'sphinx_book_theme'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
-# html_theme_options = {}
 
-# Add any paths that contain custom themes here, relative to this directory.
-html_theme_path = [edx_theme.get_html_theme_path()]
+html_theme_options = {
+
+    "repository_url": "https://github.com/edx/edx-arch-experiments",
+    "repository_branch": "main",
+    "path_to_docs": "docs/",
+    "use_repository_button": True,
+    "use_issues_button": True,
+    "use_edit_page_button": True,
+    # Please don't change unless you know what you're doing.
+    "extra_footer": """
+        <a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/">
+            <img
+                alt="Creative Commons License"
+                style="border-width:0"
+                src="https://i.creativecommons.org/l/by-sa/4.0/80x15.png"/>
+        </a>
+        <br>
+        These works by
+            <a
+                xmlns:cc="https://creativecommons.org/ns#"
+                href="https://openedx.org"
+                property="cc:attributionName"
+                rel="cc:attributionURL"
+            >Axim Collaborative</a>
```

Comment:
Is this appropriate for edx-arch-experiments?

Reply:
(same question for license)

Reply:
Hey @timmc-edx, you are absolutely right about the Author, license and copyright information. Can I please have the exact data that we need to now add in license, author and copyright against this repo?

Reply:
I heard back from Legal -- we should use `edX LLC` for copyright. As far as licensing, I believe it's correct to leave the docs at cc-by-sa-4, because docs.edx.org uses that in its footer. I'll check in with legal on that asynchronously, though. (It's probably not correct to have the repo licensed as AGPL or Apache but the published docs as CC, but that's probably going to be a longer conversation...)